### DISCUSSION TITLE
How to customize icons for logout usermenu?

### DISCUSSION THREAD

**Discussion by fahmiSD** [farridav/django-jazzmin#482] [Q&A]
Hello, i love jazzmin theme...
but want to customize logout icons in user menu. what i know is we can use this line :
`"usermenu_links" : {}`

but i don't know how to pointing to logout button, any idea?

![image](https://user-images.githubusercontent.com/21198221/219270568-cbeb1c9c-f508-4ef2-8a33-3392bfb40b78.png)

**Comment by gabriel-albqr** [comment]
You can customize the logout button in the user menu using `usermenu_links` in your `JAZZMIN_SETTINGS`.

Example:

```python
JAZZMIN_SETTINGS = {
    "usermenu_links": [
        {"name": "Change password", "url": "admin:password_change", "icon": "fas fa-key"},
        {"name": "Log out", "url": "admin:logout", "icon": "fas fa-sign-out-alt"},
    ]
}
